File: 程式語言 C V2答案檔/109.c
#include<stdio.h>
#include<stdlib.h>

int main(){
	int score;
	
	scanf("%d", &score);
	
	if(60 <= score && score <= 100){
		printf("pass\n");
		if(score % 2 == 0){
			printf("even");
		}
		else{
			printf("odd");
		}
	}
	else if(score > 100 || score < 0 ){
		printf("error");
	}
	else{
		printf("fail\n");
		if(score % 2 == 0){
			printf("even");
		}
		else{
			printf("odd");
		}
	}
}

File: Programming Language(ver.2)/PHP/109.php
<?php
    $a = doubleval(fgets(STDIN));
    if($a > 100 || $a < 0){
        printf("error\n");
        return;
    }
    printf("%s\n", $a >= 60 ? "pass" : "fail");
    printf("%s\n", $a % 2 == 0 ? "even" : "odd");
?>